specification/appconfiguration/AppConfiguration/client.tsp

```diff
@@ -8,7 +8,8 @@ using AzureAppConfiguration;
 namespace SdkCustomizations;
 
 // Cross-language customizations
-@@clientName(Foundations.Error, "ErrorDetail");
+// except Java, since Java has built-in type in azure-core for Error model.
+@@clientName(Foundations.Error, "ErrorDetail", "python,csharp,javascript,go");
```

Comment:
I dont believe this decorator supports comma separated languages. @iscai-msft can you confirm?

I think it needs to be this like:
```
@@clientName(Foundations.Error, "ErrorDetail", "python");
@@clientName(Foundations.Error, "ErrorDetail", "csharp");
```

Reply:
`scope` supports comma separated scopes. E.g. https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-specs/http/client/structure/client-operation-group/client.tsp#L52-L65